api/variables.go

```diff
@@ -12,8 +12,16 @@ import (
 )
 
 const (
-	ErrVariableNotFound     = "variable not found"
-	ErrVariableMissingItems = "variable missing Items field"
```

Comment:
note: this wasn't used so ... should be safe to just remove?

Reply:
I checked in CT's [`nomad_var_get.go`](https://github.com/hashicorp/consul-template/blob/main/dependency/nomad_var_get.go) and [`nomad_var_list.go`](https://github.com/hashicorp/consul-template/blob/main/dependency/nomad_var_list.go#L80) and found no consumers either. _Technically_ breaking backcompat but I seriously doubt there are any real consumers.